apps/prairielearn/src/ee/pages/administratorInstitutionCourse/administratorInstitutionCourse.html.ts

```diff
@@ -26,10 +27,9 @@ export function AdministratorInstitutionCourse({
     <!doctype html>
     <html lang="en">
       <head>
-        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/head')%>", {
-          ...resLocals,
-          navPage: 'administrator_institution',
-          pageTitle: 'Courses',
+        ${HeadContents({
+          resLocals,
+          pageTitle: `Institution Administration - ${course.short_name}`,
```

Comment:
This makes sense for most of the pages in this PR. However, one minor nit, these pages that have titles with "Institution Admin...", the title gets pretty long and some of the information can be hidden or tough to display. In a Chrome browser this tab now looks like this: 

![Screenshot 2024-07-22 at 11 07 55 AM](https://github.com/user-attachments/assets/45925684-f12c-4dfa-8cff-8ba7ad56fd0d)

Whereas the previous way was a little more concise and better fit a Chrome tab:
![Screenshot 2024-07-22 at 11 07 16 AM](https://github.com/user-attachments/assets/56f26c54-7ade-4ac8-8778-47eda1f7531e)

I also previewed in Safari and this not an issue as the tabs are larger. This is pretty minor, so I don't want to block on it but something I at least wanted to point out.

Reply:
I changed the order so that the most specific info comes before and is easier to see in the title. I think it's important to have a relevant title also for bookmarks, as "Courses" does not actually provide relevant info of what the page represent.